src/pages/Account/Account.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+import User from '@services/User/User.service';
+import * as Styled from '@layout/Layout.styles';
+import { isSignedIn, getUserId } from '@utils/login';
+import { useNavigate } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { UserDataProps } from '@services/User/User.types';
+import AccountForm from '@components/AccountForm';
+
+const Account: FC = () => {
+  const navigate = useNavigate();
+  const [user, setUser] = useState<UserDataProps>();
+
+  useEffect(() => {
+    const loadData = async () => {
+      let userId = getUserId();
+      setUser((await User.get(userId)).data);
+    };
+    if (!isSignedIn()) {
+      navigate('/login');
+    } else {
+      loadData();
+    }
+  }, []);
+
+  const onEdit = (user: UserDataProps) => {
+    User.update(getUserId(), user)
+      .then((response) => {
+        console.log({ response });
+        alert('Changes saved.');
+      })
+      .catch((error) => {
+        alert(error);
+      });
+  };
+  const onPasswordChange = (newPassword: string) => {
+    User.updatePassword(getUserId(), newPassword)
+      .then((response) => {
+        console.log({ response });
+        alert('Changes saved.');
+      })
+      .catch((error) => {
+        alert(error);
+      });
+  };
+
+  let form;
+  if (user) {
```

Comment:
I would make it as:
`
return <Styled.Wrapper>{user && <AccountForm .. />}</ Styled.Wrapper>
`

Reply:
Addressed in commit: [03298211be1510af94c17aaecbe74406efdebccf](https://github.com/Luvkitri/zpab-frontend/pull/2/commits/03298211be1510af94c17aaecbe74406efdebccf)